app/controllers/sessions_controller.rb

```diff
@@ -6,16 +6,26 @@ def new
   def create
     user = User.find_by(email: params[:session][:email].downcase)
     if user && user.authenticate(params[:session][:password])
-      log_in user
-      redirect_to user_url(user)
-    else
-      flash[:danger] = 'Invalid email/password combination'
-      render 'new'
+      if user.activated? 
+        log_in user
+        params[:session][:remember_me] == '1' ? remember(user) : forget(user)
```

Comment:
what's the point of forgetting a user in this instance? seems like remembering one is the only valuable action to be taken here